packages/components/src/navigator/navigator-back-button/component.tsx

```diff
@@ -23,5 +23,5 @@ function UnconnectedNavigatorBackButton(
 
 export const NavigatorBackButton = contextConnect(
 	UnconnectedNavigatorBackButton,
-	'NavigatorBackButton'
+	'Navigator.BackButton'
```

Comment:
Double-checking if this is OK - I found it unexpected that here we use the `Navigator.BackButton` context name, but in the button hook, we use `NavigatorBackButton` without the dot notation:

https://github.com/WordPress/gutenberg/blob/fdff00c9b442dd4263da62a7dd8a3f9fe617b3ad/packages/components/src/navigator/navigator-back-button/hook.ts#L23

Same for the rest of the subcomponents and their context.

Reply:
Nope, that's a bug — well spotted! I'm going to push an update

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/64613/commits/d153bcf0db769fb57c960c90301e09a2ba24c6c2